src/plugins/dashboard/public/dashboard_app/top_nav/controls_toolbar_button/controls_toolbar_button.tsx

```diff
@@ -27,9 +28,21 @@ export function ControlsToolbarButton({ controlGroup }: { controlGroup: ControlG
       {({ closePopover }: { closePopover: () => void }) => (
         <EuiContextMenuPanel
           items={[
-            <AddDataControlButton controlGroup={controlGroup} closePopover={closePopover} />,
-            <AddTimeSliderControlButton controlGroup={controlGroup} closePopover={closePopover} />,
-            controlGroup.getEditControlGroupButton(closePopover),
+            <AddDataControlButton
+              key="addControl"
+              controlGroup={controlGroup}
+              closePopover={closePopover}
+            />,
+            <AddTimeSliderControlButton
+              key="addTimeSliderControl"
+              controlGroup={controlGroup}
+              closePopover={closePopover}
+            />,
+            <EditControlGroupButton
+              key="manageControls"
+              controlGroup={controlGroup}
+              closePopover={closePopover}
+            />,
```

Comment:
This is better style IMO + it makes the fix in https://github.com/elastic/kibana/pull/153320/files#r1142698662 possible